src/sentry/plugins/base/bindings.py

```diff
@@ -28,11 +26,15 @@ def all(self):
 
 
 class RepositoryProviderManager(ProviderManager):
-    type = providers.RepositoryProvider
+    from sentry.plugins.providers import repository
```

Comment:
Does this work? I'd assume that just imports straight away while loading the class definition

Reply:
Why wouldn't loading the class definition block the import?

Reply:
I just assumed that we load the module, which will load the class, which will call this import straight away. Maybe my understanding isn't quite right though?